tests/e2e/e2e_common.go

```diff
@@ -228,6 +228,15 @@ var (
 	envTestbedInfoJsonPath = "TESTBEDINFO_JSON"
 )
 
+// Config secret testuser credentials
+var (
+	configSecretTestUser1Password = "VMware!23"
+	configSecretTestUser1         = "testuser1"
+	configSecretTestUser2         = "testuser2"
+	vCenterUIUser                 = "administrator@vsphere.local"
+	vCenterUIPassword             = "Admin!23"
```

Comment:
this can be different if we are using a pipeline which creates users with minimal permissions, we can get these values from the e2e config, no need of these vars IMO